controllers/contentlibrary/v1alpha2/utils/constants.go

```diff
@@ -13,4 +13,7 @@ const (
 
 	ContentLibraryItemVmopFinalizer        = "contentlibraryitem.vmoperator.vmware.com"
 	ClusterContentLibraryItemVmopFinalizer = "clustercontentlibraryitem.vmoperator.vmware.com"
+
+	TKGServiceTypeLabelKeyPrefix        = "type.services.vmware.com/"
+	MultipleCLServiceTypeLabelKeyPrefix = "services.supervisor.vmware.com/"
```

Comment:
Please add a comment to this constant that indicates why, where, when, and how it will be used.